xarray/backends/common.py

```diff
@@ -369,10 +369,32 @@ class BackendEntrypoint:
     - ``guess_can_open`` method: it shall return ``True`` if the backend is able to open
       ``filename_or_obj``, ``False`` otherwise. The implementation of this
       method is not mandatory.
+
+    Attributes
+    ----------
+
+    open_dataset_parameters : tuple, default None
+        A list of ``open_dataset`` method parameters.
+        The setting of this attribute is not mandatory.
+    description : str
+        A short string describing the engine.
+        The setting of this attribute is not mandatory.
+    docs : str
+        A string with the URL to the backend's documentation.
+        The setting of this attribute is not mandatory.
     """
 
     open_dataset_parameters: tuple | None = None
-    """list of ``open_dataset`` method parameters"""
+    description: str = ""
+    docs: str = ""
+
+    def __str__(self) -> str:
+        txt = f"Backend type: {type(self).__name__}."
+        if self.description:
+            txt += f"\n{self.description}"
+        if self.docs:
+            txt += f"\nFind more info at {self.docs}"
```

Comment:
```suggestion
            txt += f"\nRead more at {self.docs}"
```
minor: suspect minimizing length here is a good thing.

Shall we call it `url` instead of `docs` so that it's clear what's expected

Reply:
> Shall we call it `url` instead of `docs` so that it's clear what's expected

I've wondered this too - I'll make that change.